### ISSUE TITLE
Support for Elrond / eGold and eGold Staking

### ISSUE THREAD

**Issue by osirion** [rotki/rotki#2789]
## Abstract

Add support for the Elrond blockchain

## Motivation

This would allow tracking of value of EGLD assets and EGLD which is currently locked up in staking. Currently this is excluded from Rotki's portfolio view.

**Comment by loopinitcrypto** [comment]
Yeah also swaps aren't recognised easily when downloading the csv from the elrond explorer.
I might have to try and do it manually.

**Comment by LefterisJP** [comment]
> Yeah also swaps aren't recognised easily when downloading the csv from the elrond explorer. I might have to try and do it manually.

We have no support for this chain explicitly so CSV would not be recognized. The only thing you can do at the moment is use the generic CSV importer: https://rotki.readthedocs.io/en/latest/usage_guide.html#rotki-generic-import

Which will need manual work
